zyte_spider_templates/utils.py

```diff
@@ -4,4 +4,4 @@
 
 
 def get_domain(url: str) -> str:
-    return re.sub(r"www.*?\.", "", parse_url(url).netloc)
+    return re.sub(r"^(www)\d?\.", "", parse_url(url).netloc)
```

Comment:
```suggestion
    return re.sub(r"^www\d*\.", "", parse_url(url).netloc)
```

The parenthesis were unnecessary, and `*` works for www54.example.com as well (if we want that).

Reply:
Good call!